src/main/resources/io/cryostat/net/openshift/OpenShiftAuthManager.properties

```diff
@@ -0,0 +1,13 @@
+# Maps a ResourceType to a Kubernetes resource. See OpenShiftAuthManager.GroupResource.
+
+# Kubernetes resource kinds:
+# DEPLOYMENTS="apps/deployments"
+# PODS="pods"
+# CRYOSTATS="operator.cryostat.io/cryostats"
+# FLIGHTRECORDERS="operator.cryostat.io/flightrecorders"
+# RECORDINGS="operator.cryostat.io/recordings"
+
+# ResourceType to Kubernetes resource kind mapping:
+RECORDING=operator.cryostat.io/recordings
+CERTIFICATE=apps/deployments,pods
+
```

Comment:
For the syntax, how about instead of `group/resource` we use `resource.group`? This matches what you would see with `kubectl auth can-i --list`. The resource will always be the substring up This has the advantage of allowing us to add support for subresources like `pod/exec` or `deployments.apps/scale`, which use the slash as a separator.

Reply:
Makes sense, thanks. I've pushed one extra commit that changes the format to `resource.group/sub`

Reply:
Do we need to definitions for the other Cryostat resource types here (e.g. `TARGET`, `CREDENTIALS`, etc.)?

Reply:
Sure, that's what this PR aims to make easier :-)

Do you have any suggestions for what k8s RBAC types we should map those application-level resources to? ie if a user wants to `CREATE TARGET`, aka create a custom target definition, what permissions should their OpenShift account have?

Reply:
Would `CREATE TARGET` have some effect on the target JVM? Like Cryostat establishing a remote JMX connection for example?

Reply:
No, not until some client tries to do something like start a recording on that target.

Reply:
> What if we just add the existing set of permissions to the properties file?

Sure - you mean to just transfer the pre-existing code-defined permissions map (around here: https://github.com/cryostatio/cryostat/blob/b44770868ca2ef66991404a03c8e161e60012e90/src/main/java/io/cryostat/net/openshift/OpenShiftAuthManager.java#L518) into the `.properties`?

Reply:
Right

Reply:
Done in the latest commit